<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sports</class>
 <widget class="QDialog" name="Sports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>运动管理</string>
  </property>
  <widget class="QPushButton" name="btnRecord">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>301</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStats">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>301</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>统计</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressDistance">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>22</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressCount">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>56</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
